test/e2e/host_path.go

```diff
@@ -87,17 +67,25 @@ var _ = Describe("hostDir", func() {
 			fmt.Sprintf("--rw_new_file=%v", filePath),
 			fmt.Sprintf("--file_mode=%v", filePath),
 		}
-		testContainerOutputInNamespace("emptydir r/w on tmpfs", c, pod, 0, []string{
+		f.TestContainerOutput("hostdir r/w on tmpfs", pod, 0, []string{
```

Comment:
Same thing as above; this doesn't really have anything to do with tmpfs.  Rename the `It` argument to something like 'should support r/w' and take the reference to tmpfs out of the first argument to `TestContainerInOutput`.